web/src/settings_account.ts

```diff
@@ -909,4 +938,26 @@ export function set_up(): void {
             $("#account-settings .privacy-setting-status").expectOne(),
         );
     });
+
+    // Set the default role of the user.
+    const current_user_role = people.get_by_user_id(current_user.user_id).role;
+    $("#user-self-role-select").val(current_user_role);
```

Comment:
This can be removed now since `update_user_self_role_select_state` handles that.